### PR TITLE
Add html, css file and linters config

### PR DESCRIPTION
**Pull request by medijdir**
# 📗 Table of Contents:
In this project 💻, I have used 👍 : 
- HTML Technologies
- CSS  
- GitHub flow
-  Linters config

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -55,7 +56,18 @@ h1 {
...
+
+.works {
+  width: 100%;
+  height: 552px;
+  padding: 16px;
+  border-radius: 16px;
+  border: 1px;
+  gap: 12px;
+  background-color: #fff;
+  margin-bottom: 25%;
```
Comment (Cabraham1) [inline style.css]: - nice work so far 👍🏿 please as per project requirement you are to display your work  card using the grid system lay pleases  do well to implement this changes

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @medijdir   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
- [ ] I kindly request that you adhere to the Figma design provided for your project. The Figma design serves as a visual reference and guideline for implementing the desired user interface and overall look of your application. Please ensure that the implemented code aligns with the design specifications provided in Figma. This includes aspects such as layout, color scheme, typography, and any other visual elements defined in the design.

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -79,3 +92,252 @@ h1 {
...
+
+.btn {
+  width: 126px;
+  height: 48px;
+  top: 12px;
+  padding: 12px;
+  border-radius: 8px;
+  gap: 10px;
+  margin: 12px 0;
+}
```
Comment (Alusp) [inline style.css]: - [ ] Good job in implementing the changes requested by the Previous Reviewer, However, as part of the requirement that says: `You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&mode=design&t=ZwtntAgqTPwDdRBG-0) and as the Previous Reviewer mentioned above, please ensure that your design matches the given design. matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is the Screenshot showing the difference: :arrow_down: 

<table>
    <tr>
      <td>Your design </td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

 
Student width

</td>
      <td>

figma Design width

</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @medijdir, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


close

 ## Highlights

- [ ] Linters set-up ✔️
- [ ] Commit messages descriptive ✔️
- [ ] Professional Github Flow used ✔️
- [ ] Professional PR Summary Flow used ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (medijdir) [reply style.css]: hi @Alusp and @Cabraham1 ,
Thanks for your help and your good work, please I have made it my better to stick to the Figma design. I have fixed the last print screen issue and this is my last review that is why kindly I need your help to tell me if there are other issues to fix it.

File: index.html [hunk 3]
```diff
@@ -33,8 +35,146 @@ <h1>I’m Mehdi jdir. <br>
...
+                    <li class="bouton b1">HTML</li>
+                    <li class="bouton b2">CSS</li>
+                    <li class="bouton b3">JavaScript</li>
+                </ul>
+                <button class="btn"><p class="btn_txt">See project</p></button>
+            </div>
+        </section>
+    </section>
+
+        <section class="About">
```
Comment (wayungi) [inline index.html:134]: - [ ] Please note that the design has a curved top right corner. Kindly implement this. 
- [OPTIONAL] I suggest that you open the design and compare it with your implementation side by side to find out which parts you have not matched to the design yet. 

des1

File: style.css (hunk 2 above)
Comment (wayungi) [reply style.css]: Hi @medijdir, You still have 3 extra reviews to use. These are more than enough to get your project approved if you commit to implementing the changes requested. 😄

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @medijdir , 

image

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
Hi @medijdir 👋
 
# STATUS: APPROVED 😃
git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Understand how to parse a Figma design to create a UI.
- ✔️ Use Flexbox to place elements on the page.
- ✔️ Use CSS Grid to place elements on the page.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medijdir** [comment]
hey @Alusp @Cabraham1  @abdulhamiid  @wayungi  🤝🤝🤝🤝🎉🎈🪄🎉👌
NyMetsJeffMcneilGIF
Thanks for your help and time.

**Pull request merged**